packages/roosterjs-editor-plugins/lib/plugins/TableResize/TableResize.ts

```diff
@@ -506,6 +512,10 @@ export default class TableResize implements EditorPlugin {
     }
 
     private frameAnimateResizeCells = (e: MouseEvent) => {
+        if (!this.currentTable.hasAttribute('isPreProcessed')) {
```

Comment:
I think we can just preprocess it every time, no need to have "isPreProcessed".

Reply:
good point

Reply:
I tried with preprocess every time with the current change, it would set table width each time, causing some jumpy effect when we resize each column, let's keep it as it is.